aws/table_aws_efs_file_system.go

```diff
@@ -58,6 +58,12 @@ func tableAwsElasticFileSystem(_ context.Context) *plugin.Table {
 				Description: "The time that the file system was created.",
 				Type:        proto.ColumnType_TIMESTAMP,
 			},
+			{
+				Name:        "automatic_backup",
```

Comment:
```suggestion
				Name:        "automatic_backups",
```

I think this column name is a bit better, as the feature is usually described as `automatic backups`.